lib/plugins/aws/package/compile/events/msk/index.test.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+const chai = require('chai');
+const runServerless = require('../../../../../../../test/utils/run-serverless');
+
+const { expect } = chai;
+
+chai.use(require('chai-as-promised'));
+
+describe('AwsCompileMSKEvents', () => {
+  const arn = 'arn:aws:kafka:us-east-1:111111111111:cluster/ClusterName/a1a1a1a1a1a1a1a1a';
+  const topic = 'TestingTopic';
+  const enabled = false;
+  const startingPosition = 'LATEST';
+  const batchSize = 5000;
+
+  describe('when there are msk events defined', () => {
+    let minimalEventSourceMappingResource;
+    let allParamsEventSourceMappingResource;
+    let defaultIamRole;
+    let naming;
+
+    before(() =>
+      runServerless({
+        fixture: 'function',
+        configExt: {
+          functions: {
+            foo: {
+              events: [
+                {
+                  msk: {
+                    topic,
+                    arn,
+                  },
+                },
+              ],
+            },
+            other: {
+              events: [
+                {
+                  msk: {
+                    topic,
+                    arn,
+                    batchSize,
+                    enabled,
+                    startingPosition,
+                  },
+                },
+              ],
+            },
+          },
+        },
+        cliArgs: ['package'],
+      }).then(({ awsNaming, cfTemplate }) => {
```

Comment:
Let's switch to _async/await_ (we've released v2 in a meantime, and ideally all new async code should be introduced via async functions)